banzai/qc/header_checker.py

```diff
@@ -0,0 +1,155 @@
+"""
+This module performs controls on several critical parameters of image header, contains in
+header_expected_format dictionnary.
+@author:ebachelet
+"""
+from banzai.stages import Stage
+from banzai import logs
+
+class HeaderSanity(Stage):
+    """
+       This class contains the needed function/definitions to performs the header check.
+    """
+
+    def __init__(self, pipeline_context):
+        super(HeaderSanity, self).__init__(pipeline_context)
+
+        self.header_expected_format = {'RA': str, 'DEC': str, 'CAT-RA': str, 'CAT-DEC': str,
+                                       'OFST-RA': str, 'OFST-DEC': str, 'TPT-RA': str,
+                                       'TPT-DEC': str, 'PM-RA': str, 'PM-DEC': str,
```

Comment:
I am nervous to include all of the variations of RA and Dec here because they may change over time. I don't really have a good solution to block against that though.